### PR TITLE
Fix part of #5070: Display empty answer message in selection interaction

### PR DESCRIPTION
**Pull request by theMr17**
## Explanation

Fixes part of #5070

Enables the `submit_answer_button` when the pending answer is empty. Instead of disabling the button, an error message, stating "_**Choose an answer to continue.**_", is now displayed when the user attempts to submit without selecting any input.

https://github.com/oppia/oppia-android/assets/84731134/ba918fb9-5857-4daf-b851-8645791e1acb

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide))
- For PRs introducing new UI elements or color changes, both light and dark mode screenshots must be included
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

### CODE REVIEW

**Comment by theMr17** [comment]
@adhiamboperes PTAL

File: app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt [hunk 1]
```diff
@@ -64,7 +78,9 @@ class SelectionInteractionViewModel private constructor(
   val choiceItems: ObservableList<SelectionInteractionContentViewModel> =
     computeChoiceItems(choiceSubtitledHtmls, hasConversationView, this, enabledItemsList)
 
+  private var pendingAnswerError: String? = null
   private val isAnswerAvailable = ObservableField(false)
+  var errorMessage = ObservableField<String>("")
```
Comment (adhiamboperes) [inline app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt]: Can be a val instead.

**Review by adhiamboperes — APPROVED** [review]
Thanks @theMr17! I only have a monor non-blocking comment on this.

File: app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt (hunk 1 above)
Comment (theMr17) [reply app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt]: Done.

**Comment by theMr17** [comment]
@adhiamboperes PTAL.

**Pull request merged**